stablehlo/dialect/StablehloOps.td

```diff
@@ -398,8 +398,9 @@ def StableHLO_IsFiniteOp: StableHLO_UnaryElementwiseOp<"is_finite", [Pure,
 }
 
 def StableHLO_LogOp: StableHLO_UnaryElementwiseOp<"log",
```

Comment:
The example used in the spec is not the same used in the ODS example. We should make them equal.